pkg/codegen/dotnet/gen.go

```diff
@@ -1146,27 +1146,26 @@ func (mod *modContext) genResource(w io.Writer, r *schema.Resource) error {
 		fun := method.Function
```

Comment:
As on other PRs, we need to pick an appropriate minimum core SDK version for the dependency we generate based on whether we use this feature.